### ISSUE TITLE
Duplicate query

### ISSUE THREAD

**Issue by nguyentranchung** [Power-Components/livewire-powergrid#60]
![Screenshot from 2021-08-31 12-00-59](https://user-images.githubusercontent.com/9611224/131434809-2e666b45-d658-4b8b-9e7d-2d64c38bd4d4.png)

```php
    public function columns(): array
    {
            Column::add()
                ->title(__('user'))
                ->field('user_name')
                ->searchable()
                ->makeInputSelect(User::get(['id', 'name']), 'name', 'user_id'),
    }
```

**Comment by luanfreitasdev** [comment]
Hello.

I'll investigate further, and if everything works out, I'll submit a PR fixing the problem

Thanks

**Comment by luanfreitasdev** [comment]
fix: [v1.2.8](https://github.com/Power-Components/livewire-powergrid/releases/tag/v1.2.8)
